global.json

```diff
@@ -5,6 +5,7 @@
   "msbuild-sdks": {
     "Microsoft.Build.CentralPackageVersions": "2.0.1",
     "Microsoft.NET.Sdk.IL": "3.0.0-preview-27107-01",
-    "Microsoft.DotNet.Arcade.Sdk": "1.0.0-beta.19270.2"
+    "Microsoft.DotNet.Arcade.Sdk": "1.0.0-beta.19306.12",
+    "Microsoft.Build.Traversal": "2.0.2"
```

Comment:
Is this adding a prebuilt?  Who's requiring this?

Reply:
I think I found a different way around this one, giving it a try without that package.